website/docs/r/vapp_vm.html.markdown

```diff
@@ -329,7 +357,8 @@ example for usage details.
 * `cpu_hot_add_enabled` - (Optional; *v3.0+*) True if the virtual machine supports addition of virtual CPUs while powered on. Default is `false`.
 * `memory_hot_add_enabled` - (Optional; *v3.0+*) True if the virtual machine supports addition of memory while powered on. Default is `false`.
 * `prevent_update_power_off` - (Optional; *v3.0+*) True if the update of resource should fail when virtual machine power off needed. Default is `false`.
-* `sizing_policy_id` (Optional; *v3.0+*, *vCD 10.0+*) VM sizing policy ID. Has to be assigned to Org VDC using `vcd_org_vdc.vm_sizing_policy_ids` and `vcd_org_vdc.default_vm_sizing_policy_id`.
+* `sizing_policy_id` (Optional; *v3.0+*, *vCD 10.0+*) VM sizing policy ID. Has to be assigned to [Org VDC](/providers/vmware/vcd/latest/docs/resources/org_vdc) using `vcd_org_vdc.vm_sizing_policy_ids` and optionally `vcd_org_vdc.default_compute_policy_id`.
+* `placement_policy_id` (Optional; *v3.8+*) VM placement policy ID. Has to be assigned to [Org VDC](/providers/vmware/vcd/latest/docs/resources/org_vdc) using `vcd_org_vdc.vm_placement_policy_ids` and optionally `vcd_org_vdc.default_compute_policy_id`.
```

Comment:
I'm comparing these docs to the elaborate PR description. Thinking that we may want to expand here for the users, e.g.: possibly worth adding sentences about how these fields relate to each other? Now it's not clear that there's an implicit relationship between them in terms of being defined and removed (here we state that both are just optional).

Reply:
Done, I've added this:
```
  In this case, if the placement policy is not set, it will pick the VDC default on creation. It must be set explicitly
  if one wants to update it to another policy (the VM requires at least one Compute Policy), and needs to be set to `""` to be removed.
```